docs/faq/migrating-from-ink-5-to-6.md

```diff
@@ -29,23 +29,9 @@ If you have an existing contract on-chain you might not be able to just
 upgrade the code on-chain, you possibly also have to migrate your storage data.
 :::
 
-## How to upgrade
-
-- Change the dependency versions of `ink` and `ink_e2e` in your contracts `Cargo.toml` to `6.0.0-alpha`.
-- Update your local `cargo-contract` installation to v6.0.0-alpha.
-- Use [`ink-node`](https://github.com/use-ink/ink-node) for local development instead of `substrate-contracts-node`.
-- Read through this page.
-
 ## Compatibility
 
-- **Rust**: >= 1.88
-- **cargo-contract**: >= v6.0.0-alpha.3
- 
-  ```bash
-  cargo install --force --locked --git https://github.com/use-ink/cargo-contract --tag v6.0.0-alpha.3
-  ```
-- **`polkadot-sdk`**: [this branch](https://github.com/use-ink/polkadot-sdk/tree/pallet-revive-with-system-and-storage-precompiles)
```

Comment:
I believe the compatible `polkadot-sdk` version is now not visible anywhere. Can you add it to the setup page?